src/features/users/PageAdminUserCreate.tsx

```diff
@@ -45,25 +48,33 @@ export default function PageAdminUserCreate() {
     },
   });
 
-  const form = useForm<UserFormFields>({
-    onValidSubmit: (values) => {
-      createUser.mutate(values);
+  const form = useForm<FormFieldUser>({
+    resolver: zodResolver(zFormFieldsUser()),
+    defaultValues: {
+      name: '',
+      email: '',
+      language: DEFAULT_LANGUAGE_KEY,
+      authorizations: ['APP'],
     },
   });
 
   return (
-    <Formiz connect={form} autoForm>
+    <Form
+      {...form}
+      onSubmit={(values) => {
+        createUser.mutate(values);
+      }}
+    >
       <AdminLayoutPage containerMaxWidth="container.md" showNavBar={false}>
         <AdminLayoutPageTopBar
-          leftActions={<AdminBackButton withConfrim={!form.isPristine} />}
+          leftActions={<AdminBackButton withConfrim={form.formState.isDirty} />}
```

Comment:
It was already here before this PR but there is a typo (in `AdminBackButton`) 
```suggestion
          leftActions={<AdminBackButton withConfirm={form.formState.isDirty} />}
```